server/database/entities/users.py

```diff
@@ -0,0 +1,24 @@
+from sqlalchemy.orm import Session
+from uuid import UUID, uuid4
+import dataclasses
+from server.database.schemas.users import UserCreate
+
+
+@dataclasses.dataclass
+class User:
+    id: UUID
+    email: str
+    hashed_password: str
+    settings: str = "{}"
```

Comment:
In an entity it's better to have settings as object

Reply:
Fixed